### DISCUSSION TITLE
Any interest in building the Android app using Github actions?

### DISCUSSION THREAD

**Discussion by lundstrj** [OpenSeizureDetector/Android_Pebble_SD#162] [03_Android App]
Since this is a fully open repo, we qualify for a fair amount of github actions. We could setup nightly builds (naturally providing there has been new code introduced to warrant a new build).

It might make it slightly more accessible for people to try non-playstore versions of OSD.

**Comment by jones139** [comment]
I have no objection to doing it - but I don't know how to!

**Reply by lundstrj** [reply]
I'm no expert but I know enough to setup nightly builds and to run tests on pull requests and such (writing those theses is usually the bigger job).
